lib/cake_pay/src/models/cake_pay_vendor.dart

```diff
@@ -17,8 +19,9 @@ class CakePayVendor {
     this.card,
   });
 
-  factory CakePayVendor.fromJson(Map<String, dynamic> json, String country) {
+  factory CakePayVendor.fromJson(Map<String, dynamic> json, String countryCode) {
     final name = stripHtmlIfNeeded(json['name'] as String);
+    final country = Country.fromCountryCode(countryCode)?.fullName ?? countryCode;
```

Comment:
the vendor already has a country, you can get it from the response instead of passing it.

and more importantly, the `unavailable` boolean was removed and replaced by `available` for better readability and easily understandable, so please change it and make sure it's handled correctly if it has any negation `!` somewhere
(note: if `available` is null, it should default to false)

Reply:
done